crates/cli-support/src/js/js2rust.rs

```diff
@@ -1,6 +1,6 @@
 use failure::Error;
 
-use super::{indent, Context};
+use super::{Context};
```

Comment:
Nitpick: when there is a single import, it is idiomatic style to drop the `{}`:

```rust
use super::Context;
```

Reply:
fixed!